MAUI/Busy-Indicator/Set-Header.md

```diff
@@ -0,0 +1,321 @@
+---
+layout: post
+title: Set Header in .NET MAUI Busy Indicator control | Syncfusion
+description: Learn here all about Set Header support in Syncfusion .NET MAUI Busy Indicator (SfBusyIndicator) control and more.
+platform: maui
+control: BusyIndicator
+documentation: ug
+---
+# Set Header in .NET MAUI Busy Indicator (SfBusyIndicator)
+
+## Set Header in SfBusyIndicator
+
+The Syncfusion Busy Indicator provides a feature to display a title with font customizations
```

Comment:
The Syncfusion Busy Indicator provides a feature to display a title with font customizations.